File: templates/Uwp/_composition/_shared/Feature.UnitTests.Core.SampleData.Grids.NUnit/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.cs
//{[{
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.NUnit
{
    public class Tests
    {
        //^^
        //{[{

        // TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [Test]
        public void EnsureSampleDataServiceReturnsGridData()
        {
            var actual = SampleDataService.GetGridSampleData();

            Assert.AreNotEqual(0, actual.Count);
        }
        //}]}
    }
}

File: templates/Uwp/_comp/_shared/Feat.UnitTests.Core.SampleData.ContentGrid.NUnit._VB/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.vb
'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Public class Tests
    '^^
    '{[{

    ' TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <Test>
    Public Sub EnsureSampleDataServiceReturnsContentGridData()
        Dim actual = SampleDataService.GetContentGridData()

        Assert.AreNotEqual(0, actual.Count)
    End Sub
    '}]}
End Class
